django/db/backends/sqlite3/features.py

```diff
@@ -46,3 +46,4 @@ class DatabaseFeatures(BaseDatabaseFeatures):
     supports_aggregate_filter_clause = Database.sqlite_version_info >= (3, 30, 1)
     supports_order_by_nulls_modifier = Database.sqlite_version_info >= (3, 30, 0)
     order_by_nulls_first = True
+    supports_bitwise_xor = False
```

Comment:
If it can be implemented in Python you attach it and expose it as `XOR` in `base.py`

e.g.

https://github.com/django/django/blob/dee0430a506f3acbc18100390b3d724720cf5e38/django/db/backends/sqlite3/base.py#L246

Reply:
Cool! I didn't know about that. We should we able to use `operator.xor`.